README.md

```diff
@@ -84,26 +84,41 @@ enum ScreenIdleState {
 Here is an example of how to use it (more detailed instructions [here](HOWTO.md)):
 
 ```js
-async function main() {
-  // feature detection.
+const main = async () => {
+  // Feature detection.
   if (!('IdleDetector' in window)) {
-    console.log("IdleDetector is not available :(");
-    return;
+    return console.log('IdleDetector is not available.');
   }
-  
-  console.log("IdleDetector is available! Go idle!");
-  
+  // Check if notifications permission is granted.
+  if ((await navigator.permissions.query({name: 'notifications'})).state !== 'granted') {
+    return console.log('Notifications permission not granted.');
+  }    
   try {
-    const idleDetector = new IdleDetector();
+    const controller = new AbortController();
+    const signal = controller.signal;
+    
+    const idleDetector = new IdleDetector({signal});
```

Comment:
The `signal` option is only needed on `start()`.

Reply:
Fixed.